server/routes/probationPractitionerReferrals/interventionProgressPresenter.ts

```diff
@@ -0,0 +1,66 @@
+import { ActionPlanAppointment, SentReferral, ServiceCategory } from '../../services/interventionsService'
```

Comment:
there's quite a bit of duplication in this module and the SP version - not a blocker, but would be nice to reuse logic like 'referralAssigned' 'hasSessions' 'tagArgs' etc

Reply:
Have added IC-1507 to look into this, as there might be some re-working we can do but I don't want to spend too much time on it now.